amethyst_ui/src/prefab.rs

```diff
@@ -11,7 +11,8 @@ use {
     TextEditing, UiButton, UiFocused, UiImage, UiText, UiTransform,
 };
 
-/// Loadable `UiTransform` data
+/// Loadable `UiTransform` data.
+/// By default z is equal to one.
```

Comment:
Why was this default chosen?

Reply:
because when defining prefabs or complex ui elements with parenting they can automatically stack on top of each other without manually defining z=1

Reply:
Ok, that's actually a pretty good reason.  It can stay :)